src/Tools/Microsoft.dotnet-openapi/src/IHttpClientWrapper.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Openapi.Tools
+{
+    public interface IHttpClientWrapper : IDisposable
```

Comment:
`internal`?

Reply:
You're right, I just haven't been being very careful about access modifiers because nobody should be using this outside of us. Will fix.